orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -65,10 +65,28 @@ public String name() {
     public TypeDescription type() {
       return type;
     }
+
+    @Override
+    public boolean equals(Object o) {
+      if (this == o) {
+        return true;
+      }
+      if (o == null || getClass() != o.getClass()) {
+        return false;
+      }
+
+      OrcField orcField = (OrcField) o;
+      return Objects.equals(name, orcField.name) && Objects.equals(type, orcField.type);
+    }
+
+    @Override
+    public int hashCode() {
+      return Objects.hash(name, type);
+    }
   }
 
-  private static final String ICEBERG_ID_ATTRIBUTE = "iceberg.id";
-  private static final String ICEBERG_REQUIRED_ATTRIBUTE = "iceberg.required";
+  static final String ICEBERG_ID_ATTRIBUTE = "iceberg.id";
+  static final String ICEBERG_REQUIRED_ATTRIBUTE = "iceberg.required";
```

Comment:
I don't see where these are used outside of this class. Why make the package-private instead of private?